doc/release_notes/release_notes_2.0.rst

```diff
@@ -0,0 +1,285 @@
+support both partition mode and sharing mode simultaneously, as shown in
+this architecture diagram:
+
+.. figure:: ../introduction/images/ACRN-V2-high-level-arch.png
+   :width: 700px
+   :align: center
+
+   ACRN V2 high-level architecture
+
+On the left, resources are partitioned and used by a pre-launched User
+Virtual Machine (VM), started by the hypervisor before the Service VM
+has been launched. It runs independent of other virtual machines, and
+can own its own dedicated hardware resources, such as a CPU core,
+memory, and I/O devices. Because other VMs may not even be aware of its
+existence, this pre-launched VM can be used as a safety VM where, for
+example, platform hardware failure detection code can run and take
+emergency actions if a system critical failure occurs.
+
+On the right, the remaining hardware resources are shared by the Service
+VM and User VMs.  The Service VM can access hardware resources directly
+(by running native drivers) and offer device sharing services to other
+User VMs by the Device Model.
+
+Also on the right, a special post-launched real-time VM (RTVM) can run a
+hard real-time OS, such as VxWorks*, Zephyr*, or Xenomai*. Because of
+its real-time capabilities, the RTVM can be used for soft PLC, IPC, or
+Robotics applications.
+
+New Hardware Platform Support
+=============================
+
+This release adds support for 8th Gen Intel® Core™ Processors (code
+name: Whiskey Lake). (See :ref:`hardware` for platform details.)
+
+Pre-launched Safety VM Support
+==============================
+
+ACRN supports a pre-launched partitioned safety VM, isolated from the
+Service VM and other post-launched VM by using partitioned HW resources.
+For example, in the hybrid mode, a real-time Zephyr RTOS VM can be
+“pre-launched” by the hypervisor even before the Service VM is launched,
+and with its own dedicated resources to achieve a high level of
+isolation. This is designed to meet the needs of a Functional Safety OS. 
+
+Post-launched VM support via OVMF
+=================================
+
+ACRN supports Open Virtual Machine Firmware (OVMF) as a virtual boot
+loader for the Service VM to launch post-launched VMs such as Windows,
+Linux, VxWorks, or Zephyr RTOS. Secure boot is also supported. 
+
+Post-launched Real-time VM Support
+==================================
+
+ACRN supports a post-launched RTVM, which also uses partitioned hardware
+resources to ensure good real-time performance, as required for
+industrial use cases.
+
+Real-Time VM Performance Optimizations
+======================================
+
+ACRN 2.0 improves (preempt-RT kernel-based) RTVM’s performance with
+these optimizations:
+
+* **Reduce the impact of VM-Exit overhead:**
+   Use Local APIC (LAPIC) passthrough, Virtio Polling Mode Drivers (PMD),
+   and NMI interrupt notification technologies. 
+
+* **Isolate the RTVM from the Service VM:**
+   Use Cache Isolation via Catch Allocation Technology (CAT) to ensure the
```

Comment:
Should we talk about RDT instead, and mention CAT as one of the RDT technologies. I believe we also have CDP (Code and Data Prioritization) and maybe others (MBM?). My understanding is that RDT is the umbrella term for all these technologies so it would make sense to start from it and enumerate the ones in that family that we support already.